index.html

```diff
@@ -35,20 +35,20 @@ <h2>
             <p>In Rzeszów there are three public open indoor climbing walls:</p>
 
             <ul>
-                <li>Climbing Wall near Technical University</li>
-                <li>Baltoro Climbing Centre</li>
-                <li>Climbing wall near Rzeszów University</li>
+                <li><a href="#climbing-wall-near-technical-university-politechnika-rzeszowska">Climbing Wall near Technical University</a></li>
```

Comment:
Isn't it change positioning of this page?

Reply:
@jhryniuk Sorry but I can't get it, can you please elaborate your question? Idea here is to get clickable list. And after click scroll to section.